DOCUMENTATION.md

```diff
@@ -48,7 +48,9 @@ MQTTClient(app_id, app_access_key, mqtt_address="", discovery_address="discovery
   ![Screenshot of the console with accesskey section](./images/accesskey-console.png?raw=true)
 * `mqtt_address`: **string** this is the address of the handler to which the application was registered, in the `{hostname}:{port}` format.
 * `discovery_address`: **string** this is the address of the discovery server to use in order to find back the address of the MQTT handler, in the `{hostname}:{port}` format.
-  The constructor returns an **MQTTClient object** set up with the application informations, ready to be connected to The Things Network.
+
+    If the `mqtt_address` is set, the `discovery_address` doesn't need to be set as it is only used to retrieve the `mqtt_address` from the discovery server. The mqtt_address will be used to open a data connection with the application.
```

Comment:
"The mqtt_address will be used to open a data connection with the application." No need to specify this